gen-windows-VS2019.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
This file should not be duplicated, have vs2015 as the default and vs2017 as an option and ask the user.